src/modules/map/components/CompareSlider.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div class="compare-slider">
+        <font-awesome-icon
+            class="compare-slider-caret-left"
+            :icon="['fas', 'caret-up']"
+            :rotation="270"
+        />
+        <div class="compare-slider-line"></div>
+
+        <font-awesome-icon
+            class="compare-slider-caret-right"
+            :icon="['fas', 'caret-up']"
+            :rotation="90"
+        />
+    </div>
+</template>
+
+<script>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+export default {
+    components: {
+        FontAwesomeIcon,
+    },
+}
+</script>
+
+<style lang="scss" scoped>
+@import 'src/scss/webmapviewer-bootstrap-theme';
+
+$slider-position: 50%;
+
+.compare-slider {
+    position: absolute;
+    top: 0;
+    display: inline-block;
+    width: 40px;
+    height: 100%;
+    left: $slider-position;
+    border-top: 10px solid transparent;
+    border-bottom: 10px solid transparent;
```

Comment:
I don't know if this is useful or not, it acts like a margin top and bottom in this case.
Why did you need this border top and bottom defined?